install.sh

```diff
@@ -164,36 +164,36 @@ _install_unsupported() {
   if ! command -v cargo &>/dev/null; then
     if ! command -v rustup &>/dev/null; then
       _install_rust_and_cargo
+      _install_with_cargo
```

Comment:
After installing cargo we still want to offer the binary install before falling back to cargo, so removing this line should do that.

Reply:
as far as I understood, the return statement should be removed as well in order to not quit the _install_unsupported(), right? [5f3ce5e](https://github.com/shuttle-hq/shuttle/pull/1610/commits/5f3ce5e3057fc9aa6af7c799a2feb2def6f32646)